lms/djangoapps/learner_dashboard/views.py

```diff
@@ -66,6 +67,7 @@ def program_details(request, program_uuid):
 
     context = {
         'program_data': program_data,
+        'courses_progress': meter.progress(programs=meter.programs, count_only=False)[0],
```

Comment:
well i think the program data is primarily used for the program data and this is just the course data.  
Assuming we didn't need the course data we could theoretically remove it from the program_data object but that doesn't seem necessary to me